### PR TITLE
Add docs for tenant operator

### PR DESCRIPTION
**Pull request by GabriFila**
# Description

This PR adds some documentation for the tenant operator

### CODE REVIEW

**Review by giorio94 — COMMENTED** [review]
Thanks @GabriFila. I left some comments, even though more related to personal feelings as the PR deals with documentation. Feel free to ignore them if you do not agree

File: operators/README.md [hunk 1]
```diff
@@ -109,6 +113,104 @@ kubectl create secret generic <secret-name> \
   --from-file=./ssh_host_key_rsa
 ```
 
+## Tenant operator
+
+The tenant operator manages users inside the Crownlabs cluster, its workflow is based upon 2 CRDs:
```
Comment (giorio94) [inline operators/README.md:118]:
```suggestion
The tenant operator manages users inside the Crownlabs cluster, its workflow is based upon 2 CRDs:
* **Tenant**: represents a CrownLabs user (i.e. a student or a professor), describing the basic contact information and the Workspaces he is granted access to.
* **Workspace**: represents the collection of related `Templates` (e.g. those belonging to the same course)
```

**Review by giorio94 — APPROVED** [review]
/lgtm

Just a minor suggestion

**Review by palexster — APPROVED** [review]

**Pull request merged**